### ISSUE TITLE
Wants to open CloudShell on file deletion

### ISSUE THREAD

**Issue by Tbohunek** [Azure/vscode-azureterraform#215]
Every time I remove a file in wsl terminal `rm file.txt`, I get a pop-up 
```
Do you want to open CloudShell?
Source: Azure Terraform (Extension)
```
Yeah, priority 0, first world problem... Happy to help you fix it though. :)

**Comment by incorrupt** [comment]
Same things happens when a CLI ran from the vs code terminal removes a file.
This is really annoying.

**Comment by Falven** [comment]
Yeah these popups are very annoying. Please add a setting to disable them.

**Comment by ErikHumphrey** [comment]
There's no setting to suppress this prompt yet, right?

**Comment by thejohnbackes** [comment]
This is the codepath:

https://github.com/Azure/vscode-azureterraform/blob/abddbec09ccaa8041efefb80ed5bdc227aa9344f/src/cloudShell.ts#L177
https://github.com/Azure/vscode-azureterraform/blob/abddbec09ccaa8041efefb80ed5bdc227aa9344f/src/cloudShell.ts#L53

It was added by @njuCZ - @njuCZ can you comment on this issue?

**Comment by bergmeister** [comment]
still happening and so annoying that I want to disable the extension
